pycaret/loggers/comet_logger.py

```diff
@@ -0,0 +1,68 @@
+from pycaret.loggers.base_logger import BaseLogger
+from pathlib import Path
+import uuid
+from copy import deepcopy
+import joblib
+
+try:
+    import comet_ml
+except ImportError:
+    comet_ml = None
+
+class CometLogger(BaseLogger):
+    def __init__(self) -> None:
+        if comet_ml is None:
+            raise ImportError(
+                "CometLogger requires Comet. Install using `pip install comet_ml`"
+            )
+        super().__init__()
+        self.run = None
+    
+    def init_experiment(self, exp_name_log, full_name=None, **kwargs):
+        self.run = comet_ml.Experiment(project_name=exp_name_log, **kwargs)
+        self.run.set_name(full_name)
+        self.run.log_other('Created from', 'pycaret')
+        return self.run
+    
+    def log_params(self, params, model_name=None):
```

Comment:
If model name is provided here I think we can add it as a prefix to the logged parameters. 
e.g
```
self.run.log_parameters(params, prefix=model_name)
```